src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router';
 import Banner from '../../common/Banner';
 import ParentSidebar from '../ParentHome/ParentSidebar';
+import CreateNewStudent from './CreateNewStudent';
 import '../../../styles/ParentStyles/index.less';
 
 const ParentFamilyHome = () => {
+  const [studentInfo, setStudentInfo] = useState(null);
+  const [modal, setModal] = useState(false);
+
+  // useEffect(() => {
+  //   const token = JSON.parse(localStorage.getItem(`okta-token-storage`));
+  //   const config = {
+  //     headers: { Authorization: `Bearer ${token.idToken.value}` },
+  //   };
+  //   axios
+  //     .get(`https://coder-heroes-api.herokuapp.com/parent/1/children`, config)
```

Comment:
When we implement these `axios` calls, let's make sure we're using our ENV variables - we should never be hardcoding endpoints, and for development, we should be using our locals, to begin with!